packages/wonder-blocks-tooltip/.npmignore

```diff
@@ -0,0 +1,2 @@
+index_stories.js
+index_test.js
```

Comment:
I wonder if we can pass the top-level .npmignore to `lerna` as part of the publish process.  No action required.

Reply:
🤔 That would be cool